addons/material-forum-editor-buttons/userstyle.css

```diff
@@ -0,0 +1,131 @@
+@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200");
```

Comment:
I'm not sure if Chrome Web Store policy allows this

Reply:
Any "choose a custom font" implementation would do something like this, however. It's kind of a gray area.

Reply:
Oh. Any better solutions?

Earlier, I'd added the woff2 file directly, but @GrahamSH-LLK suggested I change it.
And well, he's right - the font file is huge!

Reply:
I guess the best solution would be an SVG file for each icon, included in the source of the extension.

Reply:
> Any "choose a custom font" implementation would do something like this, however. It's kind of a gray area.

Shouldn't google be fine with it? It's their own icon font from their own api.